WordPress/Classes/Categories/NSDate+StringFormatting.m

```diff
@@ -11,14 +11,35 @@ - (NSString *)shortString
                                                      toDate:[NSDate date]
                                                     options:0];
 
-    if (dateComponents.day > 0) {
-        shortString = [NSString stringWithFormat:NSLocalizedString(@"%id", @"Days duration single character abbreviation"), dateComponents.day];
-    } else if (dateComponents.hour > 0) {
-        shortString = [NSString stringWithFormat:NSLocalizedString(@"%ih", @"Hours duration single character abbreviation"), dateComponents.hour];
-    } else if (dateComponents.minute > 0) {
-        shortString = [NSString stringWithFormat:NSLocalizedString(@"%im", @"Minutes duration single character abbreviation"), dateComponents.minute];
+    NSInteger day = ABS(dateComponents.day);
+    NSInteger hour = ABS(dateComponents.hour);
+    NSInteger minute = ABS(dateComponents.minute);
+    NSInteger second = ABS(dateComponents.second);
+
+    if (day > 0) {
+        if (dateComponents.day < 0) {
```

Comment:
It took me a bit to realize why this changed, and what `ABS` does here.

Perhaps it'd be more readable if a `BOOL isFutureDate` is calculated before.

https://gist.github.com/koke/85fd7f40386ae2258c7a